src/data-workspace/inputs/generic-input.js

```diff
@@ -27,7 +27,7 @@ export const GenericInput = ({
     onFocus,
     disabled,
 }) => {
-    const [lastSyncedValue, setLastSyncedValue] = useState()
+    const [lastSyncedValue, setLastSyncedValue] = useState('')
```

Comment:
lastSyncedValue this currently the cause of a couple of issues. Just trying to see what I can do about that

Reply:
Good observation. We need to initialise the  `lastSyncedValue` with `input.value`